resources/public/api/conf/6.0/schemas/retrieve_foreign_property/def1a/retrieve_foreign_property.json

```diff
@@ -0,0 +1,1378 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "For TY 2024-25 and after",
```

Comment:
Should be just "For TY 2024-25" (as it is only for that year now that the 2025-26 schema has been added).

Reply:
Done